app/controllers/collections_controller.rb

```diff
@@ -89,6 +89,14 @@ def works
     )
   end
 
+  def history
+    authorize! @collection
+
+    @events = Sdr::Event.list(druid: @collection.druid).map { |event| EventPresenter.new(event:) }
+
+    render 'works/history', layout: false
```

Comment:
a little odd to have the works/history view render a collection history (though I understand it's basically the same, so whatever)...confused me temporarily